src/app/organization/generate/generate.component.html

```diff
@@ -0,0 +1,48 @@
+<div class="card">
+  <form enctype="multipart/form-data" method="POST">
+
+    <div class="styled-input">
+      <h5>Photo:</h5>
+      <input type="file" required/>
+    </div>
+
+    <div class="styled-input">
+      <input id="company" type="text" required/>
+      <label for="company">Company</label>
+    </div>
+
+    <div class="styled-input">
+      <input id="title" type="text" required/>
+      <label for="title">Title</label>
+    </div>
+
+    <div class="styled-input">
+      <input id="name" type="text" required/>
+      <label for="name">Name</label>
+    </div>
+
+    <div class="styled-input">
+      <input id="description" type="text" required/>
+      <label for="description">Description</label>
+    </div>
+
+    <div class="styled-input">
+      <input id="date" type="date" required/>
+      <label for="date"></label>
+    </div>
+
+    <div class="styled-input">
+      <h5>Signature:</h5>
+      <input type="file" required/>
+    </div>
+
+    <div class="btn">
+      <input type="submit" value="Preview">
+    </div>
+
+    <div class="btn">
+      <input type="button" value="Publish">
+    </div>
+
+  </form>
```

Comment:
No validations? Please add validations for mandatory fields namely name, description etc

Reply:
Validation should be placed in component.ts files and not html files